source/documentation/archive/serving_data/fuseki1.md

```diff
@@ -0,0 +1,450 @@
+
+This details the service URIs for Fuseki:
+
+-   `http://*host*/`*dataset*`/query` -- the SPARQL query endpoint.
+-   `http://*host*/`*dataset*`/update` -- the SPARQL Update language
+    endpoint.
+-   `http://*host*/`*dataset*`/data` -- the SPARQL Graph Store Protocol
+    endpoint.
+-   `http://*host*/`*dataset*`/upload` -- the file upload endpoint.
+
+Where *dataset* is a URI path. Note that Fuseki defaults to using
+port 3030 so *host* is often *localhost:3030*.
+
+**Important** - While you may set *dataset* to be the text `dataset` this should be avoided since it may interfere with the function of the control panel and web pages.
+
+The URI
+`http://host/`*dataset*`/sparql`
+is currently mapped to `/query` but this may change to being a
+general purpose SPARQL query endpoint.
+
+## Running a Fuseki Server
+
+The server can be run with the script `fuseki-server`. Common forms
+are:
+
+    fuseki-server --mem  /DatasetPathName
+
+    fuseki-server --file=FILE /DatasetPathName
+
+    fuseki-server --loc=DB /DatasetPathName
+
+    fuseki-server --config=ConfigFile
+
+There is an option `--port=PORT` to set the port number. It
+defaults to 3030.
+
+`/DatasetPathName` is the name under which the dataset will be
+accessible over HTTP.  Please see the above section on Server URI scheme for notes regarding available URIs and choice of this name
+
+The server will service read requests only unless the `--update`
+argument is used.
+
+The full choice of dataset forms is:
+
+**Fuseki Dataset Descriptions**
+
+-   `--mem` <br /> Create an empty, in-memory (non-persistent) dataset.
+-   `--file=FILE` <br />
+    Create an empty, in-memory (non-persistent) dataset, then load FILE
+    into it.
+-   `--loc=DIR` <br />Use an existing TDB database. Create an empty one if it does not
+    exist.
+-   `--desc=assemblerFile` <br /> Construct a dataset based on the general assembler description.
+-   `--config=ConfigFile` <br /> Construct one or more service endpoints based on the
+    [configuration description](#fuseki-configuration-file).
+
+A copy of TDB is included in the standalone server. An example
+assembler file for TDB is in tdb.ttl.
+
+**Fuseki Server Arguments**
+
+-   `--help`          <br /> Print help message.
+-   `--port=*number*` <br /> Run on port *number* (default is 3030).
+-   `--localhost`     <br /> Listen only to the localhost network interface.
+-   `--update`        <br /> Allow update. Otherwise only read requests are served (ignored if a configuration file is given).
+
+## Fuseki Server starting with an empty dataset
+
+    fuseki-server --update --mem /ds
+
+runs the server on port 3030 with an in-memory dataset. It can be
+accessed via the appropriate protocol at the following URLs:
+
+-   SPARQL query: `http://localhost:3030/ds/query`
+-   SPARQL update: `http://localhost:3030/ds/update`
+-   SPARQL HTTP update: `http://localhost:3030/ds/data`
+
+The [SPARQL Over HTTP](soh.html) scripts take care of naming
```

Comment:
Not sure if important for this archived page, but I get 404 for this one. Maybe `/documentation/fuseki2/soh.html` ?